roles/common/tasks/main.yaml

```diff
@@ -37,9 +37,15 @@
         V4_CFG_NFS_SVR_PATH: "{{ tfstate.rwx_filestore_path.value }}"
       when: 
         - tfstate.rwx_filestore_path is defined
+    - name: tfstate - nfs 
```

Comment:
Why the new variables CREATE_EFS_STORAGE_CLASS and CREATE_NFS_STORAGE_CLASS? They are unneeded and don't following the naming standard.